metrics/network/remote_network/remote-networking-test-common.sh

```diff
@@ -135,8 +136,8 @@ function start_server {
 	elif [ "$runtime_server" == "cc-runtime" ]; then
 		ssh "$ssh_user"@"$ssh_address" 'DOCKER_EXE=docker; \
 			server_id=$($DOCKER_EXE ps -q); \
-			mount="mount -t ramfs -o size=20M ramfs /tmp"; \
-			server_command="$mount && iperf3 -s"; \
+			init_cmds="export TMPDIR=/dev/shm"; \
```

Comment:
you have already defined ```init_cmds``` in the line 41

Reply:
@devimc yes, however, here the command will run in another host, so I need to define again the variable

Reply:
yep, but I think you could do something like this

```
init_cmds="$init_cmds"
```

if TMPDIR change in the future, you only will change its path in the line 41

Reply:
@devimc the problem is that the commands should be using "" however that will take the variables of the host a